static/js/vue-cdr-access/src/mixins/analyticsUtils.js

```diff
@@ -1,18 +1,27 @@
 export default {
     methods: {
         pageEvent(recordData) {
-            let collection = recordData.briefObject.parentCollectionName || '';
+            this.$gtag.event('record', {
+                'event_category': recordData.briefObject.parentCollectionId,
+                'event_label': `${recordData.briefObject.title}|${recordData.briefObject.id}`
+            });
+        },
 
+        matomoPageEvent(recordData) {
+            let collection = recordData.briefObject.parentCollectionName || '';
             if (collection === '' && recordData.briefObject.type === 'Collection') {
                 collection = recordData.briefObject.title;
             }
             if (collection === '') {
                 collection = '(no collection)';
             }
 
-            this.$gtag.event('record', {
-                'event_category': recordData.briefObject.parentCollectionId,
-                'event_label': `${recordData.briefObject.title}|${recordData.briefObject.id}`,
+            window._mtm = window._mtm || [];
+            window._mtm.push({
+                recordUUID: recordData.briefObject.id,
```

Comment:
Is `recordUUID` something specified by matomo or are the keys up to us? If its up to us, I'd suggestion recordId instead, rather than specifying the algorithm used to produce the ID

Reply:
i chose the name, it's a variable i'm using in matomo which can be anything. we can change it to recordId, i just have to change it in the matomo config as well.

Reply:
I changed it to `recordId` in the matomo configs for test, QA, and prod, so we should be good to change it here in the code. just need to redeploy to test after